cluster-autoscaler/cloudprovider/openstack/examples/os-autoscaler-deployment-master.yaml

```diff
@@ -0,0 +1,51 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: os-autoscaler
+  namespace: kube-system
+  labels:
+    app: autoscaler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: autoscaler
+  template:
+    metadata:
+      namespace: kube-system
+      labels:
+        app: autoscaler
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      securityContext:
+        runAsUser: 1001
+      tolerations:
+        - key: node.cloudprovider.kubernetes.io/uninitialized
+          value: "true"
+          effect: NoSchedule
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+      serviceAccountName: os-autoscaler-account
+      containers:
+        - name: os-autoscaler
+          image: thartland/os-autoscaler:v1.0
```

Comment:
This will change before merging?

Reply:
Suggestions on where to move the image to?

Reply:
Isn't it from where current CA image hosted?

Reply:
We can change this after this has been merged and there has been a release with an image including the openstack autoscaler.